src/app/services/shared.service.ts

```diff
@@ -0,0 +1,8 @@
+import { Injectable } from '@angular/core';
+import { Subject } from 'rxjs';
+
+@Injectable()
+export class SharedService {
+    public IsUserLoggedIn: Subject<any> = new Subject<any>();
+    public IsLoginSkiped: Subject<boolean> = new Subject<boolean>();
```

Comment:
should be in camelCase? `isUserLoggedIn` and `isLoginSkiped`

Reply:
and why do we need 2? can not we decide based on any? or it contain some value per subject